drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -949,6 +1044,7 @@ static struct ieee802154_radio_api nrf5_radio_api = {
 	.tx = nrf5_tx,
 	.ed_scan = nrf5_energy_scan_start,
 	.get_time = nrf5_get_time,
+	.get_csl_acc = nrf5_get_acc,
```

Comment:
I think this function could be named in a more generic way like `get_radio_clock_accuracy` or `get_radio_timings_accuracy`. It affects not only accuracy of CSL-related operations, but each scheduled TX or RX window timings.